api/HTMLIsIndexElement.json

```diff
@@ -1,148 +0,0 @@
-{
-  "api": {
-    "HTMLIsIndexElement": {
-      "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIsIndexElement",
```

Comment:
https://developer.mozilla.org/docs/Web/API/HTMLIsIndexElement will need to be removed.